HPXMLtoOpenStudio/resources/EPvalidator.rb

```diff
@@ -87,15 +87,25 @@ def self.run_validator(hpxml_doc)
         '/HPXML/Building/BuildingDetails/Appliances/ClothesWasher' => zero_or_one, # See [ClothesWasher]
         '/HPXML/Building/BuildingDetails/Appliances/ClothesDryer' => zero_or_one, # See [ClothesDryer]
         '/HPXML/Building/BuildingDetails/Appliances/Dishwasher' => zero_or_one, # See [Dishwasher]
-        '/HPXML/Building/BuildingDetails/Appliances/Refrigerator' => zero_or_one, # See [Refrigerator]
+        '/HPXML/Building/BuildingDetails/Appliances/Refrigerator' => zero_or_more, # See [Refrigerator]
+        '/HPXML/Building/BuildingDetails/Appliances/Freezer' => zero_or_one, # See [Freezer]
         '/HPXML/Building/BuildingDetails/Appliances/Dehumidifier' => zero_or_one, # See [Dehumidifier]
         '/HPXML/Building/BuildingDetails/Appliances/CookingRange' => zero_or_one, # See [CookingRange]
 
         '/HPXML/Building/BuildingDetails/Lighting' => zero_or_one, # See [Lighting]
         '/HPXML/Building/BuildingDetails/Lighting/CeilingFan' => zero_or_one, # See [CeilingFan]
 
+        '/HPXML/Building/BuildingDetails/Pools/Pool' => zero_or_more, # See [Pool]
+        '/HPXML/Building/BuildingDetails/HotTubs/HotTub' => zero_or_more, # See [HotTub]
```

Comment:
Change to `zero_or_one`

Reply:
So, I wanted to have zero_or_more so that I could test both pool electric heater and pool gas heater using the same hpxml file...

Reply:
Seems reasonable to have two hot tubs, no?